spring-cloud-task-core/src/main/java/org/springframework/cloud/task/repository/dao/JdbcTaskExecutionDao.java

```diff
@@ -114,6 +117,19 @@
 	private static final String RUNNING_TASK_EXECUTION_COUNT_BY_NAME = "SELECT COUNT(*) FROM " +
 			"%PREFIX%EXECUTION where TASK_NAME = ? AND END_TIME IS NULL ";
 
+	private static final String LAST_TASK_EXECUTIONS_BY_TASK_NAMES =
+	"select TE2.* from (" +
+	"select MAX(TE.TASK_EXECUTION_ID) as TASK_EXECUTION_ID, TE.TASK_NAME, TE.START_TIME from (" +
+	"select TASK_NAME, MAX(START_TIME) as START_TIME" +
+	"      FROM %PREFIX%EXECUTION where TASK_NAME in (:taskNames)" +
+	"      GROUP BY TASK_NAME" +
+	") TE_MAX " +
+	"inner join %PREFIX%EXECUTION TE ON TE.TASK_NAME = TE_MAX.TASK_NAME AND TE.START_TIME = TE_MAX.START_TIME " +
+	"group by TE.TASK_NAME, TE.START_TIME" +
+	") TE1 " +
+	"inner join %PREFIX%EXECUTION TE2 ON TE1.TASK_EXECUTION_ID = TE2.TASK_EXECUTION_ID " +
+	"order by TE2.TASK_EXECUTION_ID asc";
```

Comment:
Wondering if we should use start_time, task_execution_id desc like we have for the defaults of our paginated results.   Just for consistency, thoughts?